src/otx/algorithms/common/adapters/torch/dataloaders/samplers/balanced_sampler.py

```diff
@@ -21,24 +27,40 @@ class BalancedSampler(Sampler):  # pylint: disable=too-many-instance-attributes
         dataset (Dataset): A built-up dataset
         samples_per_gpu (int): batch size of Sampling
         efficient_mode (bool): Flag about using efficient mode
+        num_replicas (int, optional): Number of processes participating in
+            distributed training. By default, :attr:`world_size` is retrieved from the
+            current distributed group.
+        rank (int, optional): Rank of the current process within :attr:`num_replicas`.
+            By default, :attr:`rank` is retrieved from the current distributed
+            group.
+        drop_last (bool, optional): if ``True``, then the sampler will drop the
+            tail of the data to make it evenly divisible across the number of
+            replicas. If ``False``, the sampler will add extra indices to make
+            the data evenly divisible across the replicas. Default: ``False``.
+        use_adaptive_repeats (bool): Flag about using adaptive repeats
```

Comment:
```suggestion
        use_adaptive_repeats (bool, optional): Flag about using adaptive repeats
```
I think it's missed.

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/2428/commits/a496a82861ea094fca937a3c06caff9aac85f3fd